vaadin-grid-flow-integration-tests/src/main/java/com/vaadin/flow/component/grid/it/PreserveOnRefreshPage.java

```diff
@@ -32,6 +39,96 @@ public PreserveOnRefreshPage() {
         grid.addComponentColumn(person -> new Span(person.getFirstName()))
                 .setHeader(new Span("header")).setFooter(new Span("footer"));
         add(grid);
+        TestGui();
```

Comment:
These tests don't belong in the PreserveonRefreshPage. This is related to sorting so that's where it should be. SortingPage also already has a grid with sorting so you should only add the detach/attach buttons there.

Reply:
Thank you for correcting me. moved the test to the right place. and my PR is already review.